p/themes/base-theme/template.css

```diff
@@ -591,12 +592,17 @@ input[type="search"] {
 
 /*=== Body */
 #global {
+	background: inherit;
```

Comment:
The various `background: inherit;` are to avoid transparency, which make it difficult to hide background elements